scripts/e2e-test.sh

```diff
@@ -5,9 +5,16 @@ set -eu
 SCRIPTDIR=$(dirname "$(realpath "$0")")
 TESTDIR=$(realpath "$SCRIPTDIR/../test/e2e")
 REPORTSDIR=$(realpath "$SCRIPTDIR/..")
-# new tests should be added before the replica_pod_remove test
+
+# List and Sequence of tests.
 #tests="install basic_volume_io csi replica rebuild node_disconnect/replica_pod_remove uninstall"
-tests="install basic_volume_io csi uninstall"
+# Restrictions:
+#   1. resource_check MUST follow csi 
+#       resource_check is a follow up check for the 3rd party CSI test suite.
+#   2. replicas_pod_remove SHOULD be the last test before uninstall
+#       this is a disruptive test.
```

Comment:
This comment is important but the _replicas_pod_remove_ test isn't in the list of tests. I wonder if this should be moved above line 10 where it is commented out? Then, when it is included, the comment can be updated.

Reply:
It should be back in, e.g. in back-to-back testing fairly soon.